docs/transpile/index.mdx

```diff
@@ -0,0 +1,42 @@
+
+
+## Transpiler Stages
+Qiskit utilizes a set of six fundamental stages in the prebuilt transpiler pipeline:
+
+1. `init` - This pass runs any initial passes that are required before we start embedding the circuit to the backend.  This typically involves unrolling custom instructions and converting the circuit to all single- and two-qubit gates. (By default this does nothing)
+1. `layout` - This pass applies a *layout*, mapping/assigning the virtual qubits in your circuit to the physical qubits of a backend.
+1. `routing` - This pass runs after a layout has been applied and will inject gates (i.e., SWAPs) in the original circuit in order to make it compatible with the backend's connectivity/coupling map.
+1. `translation` - This pass translates the gates in the circuit to the backend's basis set of instructions.
+1. `optimization` - This pass runs an optimization loop to find more efficient decompositions of your quantum circuit until a condition is met (such as a fixed depth).
+1. `scheduling` - This pass is for any hardware-aware optimizations that improves the efficiency of the executed microwave pulse instructions.
+
+ If you decide to customize your own transpilation workflow, we suggest using these stages as a guideline during development.
+
+
+
+## Default Transpilation
+The basic usage of the transpiler is to call the [`qiskit.compiler.transpile()`](../api/qiskit/compiler#compilation-routines) function.  This will generate and run one of the preset `StagedPassManager`s based on, among other options, an `optimization_level` flag which can be set to either 0, 1, 2, or 3.  Higher levels generate more optimized circuits, at the expense of longer transpilation times.
+
+- `optimization_level=0`: Just maps the circuit to the backend, with no explicit optimization.
+- `optimization_level=1`: Maps the circuit, but also does some lightweight optimizations by collapsing adjacent gates.
+- `optimization_level=2`: Medium-weight optimization, including a noise-adaptive layout and a gate-cancellation procedure based on gate commutation relationships.
+- `optimization_level=3`: Heavy-weight optimization, which in addition to previous steps, resynthesizes two-qubit blocks of gates in the circuit.
+
+If you'd like to read more about the preset pass managers or the transpiler stages, look to the section on [Transpiler Stages & Preset PassManagers](passmanagers.ipynb).
```

Comment:
```suggestion
If you'd like to read more about the preset pass managers or the transpiler stages, look to the section on [Transpiler stages and preset PassManagers](passmanagers.ipynb).
```

Reply:
After a quick google across Qiskit, I see the spacing and capitalization of passmanager (PassManager? pass manager? Pass Manager?) varies quite a bit (specifically for occurrences outside of code formatting).... thoughts on the right way to write it @javabster ?